rust/platform/triple_mappings.bzl

```diff
@@ -66,6 +66,41 @@ _SYSTEM_TO_DYLIB_EXT = {
     "unknown": ".wasm",
 }
 
+# See https://github.com/rust-lang/rust/blob/master/src/libstd/build.rs
+_SYSTEM_TO_STDLIB_LINKFLAGS = {
+    # TODO: Right now bazel cc rules does not specify the exact flag setup needed for calling out
+    # TODO: system libs, that is we dont know given a toolchain if it should be, for example,
+    # TODO: `-lxxx` or `/Lxxx` or `xxx.lib` etc.
+    # TODO: We include the flag setup as they are _commonly seen_ on various platforms
+    # TODO: with a cc_rules style override for people doing things like mingw on windows.
+    # TODO: If you are reading this ... sorry! set the env var `BAZEL_RUST_STDLIB_LINKFLAGS` to
+    # TODO: what you need for your specific setup, for example like so
+    # TODO: `BAZEL_RUST_STDLIB_LINKFLAGS="-ladvapi32:-lws2_32:-luserenv"`
+
+    "freebsd": ["-lexecinfo", "-lpthread"],
+    # TODO - This ignores musl although I am not 100% sure what bazel thinks of musl anyhow
+    # NOTE - rust stdlib `build.rs` treats android as a subset of linux, rust rules treat android
+    # NOTE - as its own system
+    "linux": ["-ldl", "-lpthread"],
+    "darwin": ["-lSystem", "-lresolv"],
+    "uwp": ["ws2_32.lib"],
+    "windows": ["advapi32.lib", "ws2_32.lib", "userenv.lib"],
+    "ios": ["-lSystem", "-lobjc", "-framework Security", "-framework Foundation", "-lresolv"],
+    "android": ["-ldl", "-llog", "-lgcc"],
+    "emscripten": [],
+    "nacl": [],
+    "bitrig": [],
+    "dragonfly": ["-lpthread"],
+    "netbsd": ["-lpthread", "-lrt"],
+    "openbsd": ["-lpthread"],
+    "solaris": ["-lsocket", "-lposix4", "-lpthread", "-lresolv"],
+    "illumos": ["-lsocket", "-lposix4", "-lpthread", "-lresolv", "-lnsl", "-lumem"],
+    "fuchsia": ["-lzircon", "-lfdio"],
+    # TODO - feature backtrace is how unwind happens ...
```

Comment:
What does this TODO means?

Reply:
Bad comment on my part.

If rust stdlib is compiled for cloudabi with the `backtrace` feature it includes `-lunwind` but if the feature is not compiled in it misses this linkage. I am not sure which is the common configuration or how we encode it as a lib dependency.

I might not matter since currently its a seemingly esoteric environment that I am pretty sure Bazel wont build for all that well, I was mostly blindly translating the stdlib `build.rs`.

Reply:
Ok then maybe remove this TODO?

Reply:
I elborated the comment a little